website/search_migration/migrate.py

```diff
@@ -130,6 +130,15 @@ def migrate_users(index, delete, increment=10000):
             es_args={'raise_on_error': False})  # ignore 404s
         logger.info('{} users marked deleted'.format(total_users))
 
+def migrate_collected_metadata(index, delete):
```

Comment:
This should be telling ES to delete everything with the CGM type before calling `bulk_update`